Dockerfile

```diff
@@ -0,0 +1,124 @@
+        bzip2-dev \
+        curl-dev \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl \
+        openssl-dev \
+        pcre2-dev \
+        zlib-dev \
+    && \
+    rm -rf "/var/cache/apk/"* && \
+    autoreconf --force --install --verbose && \
+    LIBS=-lfts ./configure \
+                           --bindir="/usr/bin" \
+                           --disable-llvm \
+                           --enable-bigstack \
+                           --enable-clamdtop \
+                           --enable-clamonacc \
+                           --enable-milter \
+                           --infodir="/usr/share/info" \
+                           --libdir="/usr/lib" \
+                           --libexecdir="/usr/libexec" \
+                           --localstatedir="/var/lib/clamav" \
+                           --mandir="/usr/share/man" \
+                           --prefix="/usr" \
+                           --sbindir="/usr/sbin" \
+                           --sysconfdir="/etc/clamav" \
+                           --with-dbdir="/var/lib/clamav" \
+                           --with-group="clamav" \
+                           --with-libcurl \
+                           --with-libjson \
+                           --with-openssl="/usr" \
+                           --with-pcre \
+                           --with-user="clamav" \
+                           --with-xml \
+                           --with-zlib \
+                           --without-iconv \
+    && \
+    make DESTDIR="/clamav" install && \
+    rm -r \
+       "/clamav/usr/include" \
+       "/clamav/usr/share/man" \
+       "/clamav/usr/lib/pkgconfig/" \
+    && \
+    find "/clamav" -type f -iname '*.la' && \
+    sed -e "s|^\(Example\)|\# \1|" \
+        -e "s|.*\(PidFile\) .*|\1 /run/clamav/freshclam.pid|" \
+        -e "s|.*\(DatabaseOwner\) .*|\1 clamav|" \
+        -e "s|^\#\(UpdateLogFile\) .*|\1 /var/log/clamav/freshclam.log|" \
+        -e "s|^\#\(NotifyClamd\).*|\1 /etc/clamav/clamd.conf|" \
+        -e "s|^\#\(ScriptedUpdates\).*|\1 yes|" \
+        -e "s|^\#\(AllowSupplementaryGroups\).*|\1 yes|" \
+        "/clamav/etc/clamav/freshclam.conf.sample" > "/clamav/etc/clamav/freshclam.conf" && \
+    sed -e "s|^\(Example\)|\# \1|" \
+        -e "s|.*\(PidFile\) .*|\1 /run/clamav/clamd.pid|" \
+        -e "s|.*\(LocalSocket\) .*|\1 /run/clamav/clamd.sock|" \
+        -e "s|.*\(TCPSocket\) .*|\1 3310|" \
+        -e "s|.*\(TCPAddr\) .*|\1 0.0.0.0|" \
+        -e "s|.*\(User\) .*|\1 clamav|" \
+        -e "s|^\#\(LogFile\) .*|\1 /var/log/clamav/clamd.log|" \
+        -e "s|^\#\(LogTime\).*|\1 yes|" \
+        -e "s|^\#\(AllowSupplementaryGroups\).*|\1 yes|" \
+        "/clamav/etc/clamav/clamd.conf.sample" > "/clamav/etc/clamav/clamd.conf" && \
+    sed -e "s|^\(Example\)|\# \1|" \
```

Comment:
If there's a use case with milter running, please add it to the requested instructions, if not -- you could drop this.

Reply:
As mentioned before, milterd is started only if the configuration variable is set to do so; by default it does not start milterd. I did describe all of this as mentioned in the readme ;)